ansible/roles/ami-ovf/tasks/main.yml

```diff
@@ -28,6 +28,22 @@
       when: ansible_virtualization_type == "virtualbox"
       command: swapon -a
 
+    - name: OVF                        | Create /.pmm-ovf file
+      when: ansible_virtualization_type == "virtualbox"
+      file:
+        path: /.pmm-ovf
+        owner: root
+        group: root
+        mode: 0600
+
+    - name: AMI                        | Create /.pmm-ami file
+      when: ansible_virtualization_type != "virtualbox"
+      file:
+        path: /.pmm-ami
+        owner: root
+        group: root
+        mode: 0600
+
     - name: Packages                   | Remove PMM2 Testing Server repository
```

Comment:
Are you sure that is PMM 1.x file?

Reply:
I'm pretty sure that file is not used for PMM 1.x builds. /cc @hors

Reply:
@Sevlin  if you need to add it for PMM 1.x you need to check [pmm/main.yml](https://github.com/Percona-Lab/percona-images/blob/master/ansible/pmm/main.yml)